src/amo/constants.js

```diff
@@ -95,7 +95,7 @@ export const API_ADDON_TYPES_MAPPING = {
   extensions: ADDON_TYPE_EXTENSION,
   themes: ADDON_TYPE_STATIC_THEME,
 };
-export const VISIBLE_ADDON_TYPES_MAPPING = Object.keys(
+export const VISIBLE_ADDON_TYPES_MAPPING: Object = Object.keys(
```

Comment:
We run into the problem where Flow can't be sure about the output of `reduce`, but I agree that having a more specific type makes sense, so I added a type but also had to add a `$FlowIgnore` because of the `reduce`.